projects/packages/autoloader/src/class-plugin-locator.php

```diff
@@ -69,6 +69,12 @@ public function find_using_option( $option_name, $site_option = false ) {
 	public function find_using_request_action( $allowed_actions ) {
 		// phpcs:disable WordPress.Security.NonceVerification.Recommended
 
+		$woo_activating_plugins = $this->find_activating_plugins_in_woo_request();
```

Comment:
It would be more consistent with the rest of the `Plugin_Locator` classes to have this as a public method called in the `Plugins_Handler` class.